src/main/java/org/team1540/robot2018/commands/wrist/HoldWristPosition.java

```diff
@@ -0,0 +1,42 @@
+package org.team1540.robot2018.commands.wrist;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.team1540.robot2018.Robot;
+import org.team1540.robot2018.Tuning;
+
+public class HoldWristPosition extends Command {
+  double setpoint;
+  private long lastExecTime;
+  private long spikeDuration;
```

Comment:
What exactly is `spikeDuration` being used for, and why not a timer?

Reply:
Fair point; a timer would be a better idea.